crates/types/src/services/executor.rs

```diff
@@ -256,6 +258,27 @@ impl TransactionExecutionResult {
     }
 }
 
+/// When storage in column:key was read, it contained this value.
+#[derive(Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct StorageReadReplayEvent {
+    /// Column in the storage, identified by id.
+    pub column: u32,
+    /// Key in the column.
+    pub key: Vec<u8>,
+    /// Value at the column:key pair. None if the key was not found.
+    pub value: Option<Vec<u8>>,
+}
+impl fmt::Debug for StorageReadReplayEvent {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("StorageReadReplayEvent")
+            .field("column", &self.column)
+            .field("key", &hex::encode(&self.key))
+            .field("value", &self.value.as_ref().map(hex::encode))
+            .finish()
+    }
```

Comment:
Do we want to decode the whole value? In the case of contract it can be a lot of bytes, maybe we want to limit it with first 16 bytes?

Reply:
Full debug logs were useful sometimes. But yep, I think it's probably better to not do that now that there's an easy way to get the full values after https://github.com/FuelLabs/fuel-core/pull/2682 is implemented